website/coursework/product-design/index.md

```diff
@@ -366,7 +375,7 @@ Source: https://www.nngroup.com/articles/wireflows
 </figcaption>
 </figure>
 
-You may find it useful to develop a set of shared assets such as the app's logo, colour scheme, font and common UI widget so your mockup looks coherent when it is built by different members of the team.
+You may find it useful to develop a set of shared assets such as the app's logo, colour scheme, font and common UI elements so your mockup looks coherent when it is built by different members of the team. Figma supports [Components](https://help.figma.com/hc/en-us/articles/360038662654-Guide-to-components-in-Figma), which is a way to build elements you can reuse across your designs. You can give Figma components properties and compose them together, similar to UI library components in React/Vue/Angular. [Untitled UI](https://www.untitledui.com/) and [Riddle UI](https://www.riddleui.com/) are Figma UI kits / design systems that contain a good amount of free design system components you can use in your prototype. Figma components, used with Figma variables, empower teams to build consistent designs.
```

Comment:
Mention new features in Figma that can be used. Introduce them to 3rd party Figma UI kits which can help them build their designs faster. This may or may not be desired, would love to hear your opinion!